doc/examples/reconst_csd.py

```diff
@@ -225,7 +227,7 @@
                              parallel=True)
 
 window.clear(ren)
-fodf_peaks = actor.peak_slicer(csd_peaks.peak_dirs, csd_peaks.peak_values, scale=1.3)
+fodf_peaks = actor.peak_slicer(csd_peaks.peak_dirs, csd_peaks.peak_values)
```

Comment:
Why are you changing the scale here?

Reply:
The same reason apply to PR #1479 in example sfm_reconst.py.